club/templates/archive.html

```diff
@@ -1,6 +1,10 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:query-title {"type":"archive","level":2} /-->
+<!-- wp:query-title {"type":"archive"} /-->
```

Comment:
I noticed on mobile this increase in font size may easily cut off the word / push beyond the width of the screen:

<img width="235" alt="Screen Shot 2022-08-11 at 3 29 35 PM" src="https://user-images.githubusercontent.com/5375500/184223528-16090992-b202-4ede-932c-d9ca2a310934.png">

Reply:
We are using the font size from the design there.